configure.ac

```diff
@@ -94,19 +93,14 @@ AC_SUBST(COCKPIT_CFLAGS)
 AC_SUBST(COCKPIT_LIBS)
 
 # bridge with optional polkit
-AC_ARG_ENABLE(polkit, AC_HELP_STRING([--disable-polkit], [Disable usage of polkit]),,enable_polkit=$autodetect_polkit)
-AC_MSG_CHECKING([whether to build with polkit])
+AC_ARG_ENABLE(polkit, AC_HELP_STRING([--disable-polkit], [Disable usage of polkit]),,enable_polkit=yes)
```

Comment:
Some small bikeshedding:

`enable_polkit` will be set to
  - `yes` if `--enable-polkit`
  - `no` if `--disable-polkit`
  - not set, if neither

so it maybe makes sense to simplify this a bit and handle the default by dropping this extra assignment here and inverting the check below to test `"$enable_polkit" != "no"` (ie: "unless explicitly disabled...")

On the other hand, that will result in an undefined variable expansion (which is not an error in configure scripts, but which you might find to be undesirable).

Take it or leave it :)